packages/braid-design-system/lib/components/Autosuggest/Autosuggest.tsx

```diff
@@ -648,7 +685,15 @@ export const Autosuggest = forwardRef(function <Value>(
 
       announcements.push(translations.suggestionInstructions);
     } else {
-      announcements.push(translations.noSuggestionsAvailableAnnouncement);
+      announcements.push(
+        noSuggestionsMessage
+          ? `${noSuggestionsMessage.text}${
+              noSuggestionsMessage.description
+                ? ` ${noSuggestionsMessage.description}`
+                : ''
+            }`
+          : translations.noSuggestionsAvailableAnnouncement,
+      );
```

Comment:
Simpler if you just `.push` both each message separately, as they're [`.join`ed below](https://github.com/seek-oss/braid-design-system/blob/d3fb85d1d4776649443ba56a007adcba2cdd4f2d/packages/braid-design-system/lib/components/Autosuggest/Autosuggest.tsx#L842).

```suggestion
      announcements.push(
        noSuggestionsMessage
          ? [
              noSuggestionsMessage.text,
              noSuggestionsMessage.description,
            ].filter(Boolean)
          : translations.noSuggestionsAvailableAnnouncement,
      );
```

Reply:
Or another suggestion, a bit more verbose but I find it easier to read:

```js
    } else if (noSuggestionsMessage) {
      announcements.push(noSuggestionsMessage.text);

      if (noSuggestionsMessage.description) {
        announcements.push(noSuggestionsMessage.description);
      }
    } else {
      announcements.push(translations.noSuggestionsAvailableAnnouncement);
    }
```

Reply:
This would result in an array inside an array, and not be "joined" as below. But i did miss the join happening further down so will simplify.

Reply:
Oops, rookie mistake. I was thinking of `.concat`'s behaviour which doesn't nest arrays.